### PR TITLE
Loading Tips Update

### PR DESCRIPTION
**Pull request by TheGreatAutismo**
Updates outdated loading tips and adds a few new ones

### CODE REVIEW

**Comment by RandomityGuy** [comment]
I'd merge but I have my biased opinion about some tips which seem to annoy me:
- "Tip: Did you know you can play Marble Blast Platinum in your browser? Check out https://marbleblast.vani.ga!" : This basically markets Vani's webport and further puts my own Haxe port (which can also be played in the browser btw, check the changelogs as to see how far it has come) into more obscurity. I just don't think it is fair to solely market it, knowing the fact that there also exists another 'webport' that *actually* has accurate physics, among other stuff that it does better.
- "Trivia: Submitting a World Record as '@everyone' on the Marble Blast Platinum Web Port used to have... Unforseen consequences.": This feels useless to me, since webport is not associated with PQ, so it does not make any sense for some PQ guy to know about what happened in the webport side.

File: Marble Blast Platinum/platinum/client/scripts/loadinggui.cs [hunk 1]
```diff
@@ -353,13 +353,12 @@
...
-$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Did you know you can play Marble Blast Platinum in your browser? Check out https://marbleblast.vani.ga!";
+$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Did you know you can play Marble Blast Platinum in your browser? Check out https://marbleblast.vaniverse.io/!";
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Check out 'Marble It Up!' on Steam!";
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Check out 'Great Marble Adventure' on Steam!";
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Check out 'Custom Levels' in the Community section of the Main Menu! You'll be able to download and play custom levels all from ingame!";
-$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Want to upload your custom levels for people to play? Head to https://marbleland.vani.ga!";
+$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Want to upload your custom levels for people to play? Try out https://marbleland.vaniverse.io/ or https://marbleblast.randomityguy.me/!";
```
Comment (RandomityGuy) [inline Marble Blast Platinum/platinum/client/scripts/loadinggui.cs:360]: Bro wrong tip updated.

File: Marble Blast Platinum/platinum/client/scripts/loadinggui.cs [hunk 2]
```diff
@@ -288,17 +289,19 @@
...
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: The Bubble Powerup stays with you at all times until it is used up or you pick up a Fireball Powerup.";
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: The Sundial Item acts the same way as the Time Travel, it's just a bit fancier.";
-$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: PlatinumQuest Time Travel Items have identifiable bands; gold bands freeze the clock and red bands add time.";
+$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: PlatinumQuest's Time Travel Items have identifiable bands; gold bands freeze the clock and red bands add time.";
+$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Marble Blast Gold and Platinum's Time Travel Items can be identified by their sand color; yellow sands freeze the clock and red sands add time.";
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: In Multiplayer, if you activate a Mega Marble and hit an Ice Shard, your marble will shrink but remain unfrozen.";
+$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Colliding with an Ice Shard as a Mega Marble will shrink your marble to its regular size instead of freezing you.";
```
Comment (thearst3rd) [inline Marble Blast Platinum/platinum/client/scripts/loadinggui.cs:304]: This seems very similar to the tip above, just not multiplayer-specific. Is it meant to replace it?

File: Marble Blast Platinum/platinum/client/scripts/loadinggui.cs [hunk 3]
```diff
@@ -344,11 +347,22 @@
...
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: If you enter a Cannon as a Mega Marble, it will shrink your marble back down to size. We can't have you clogging the cannons!";
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Join our official Discord server to interact directly with the community, get links to live streams, and access new user-created content faster than ever! You can find a link to it at marbleblast.com/webchat or from the community tab only if you're logged in.";
-$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Colliding with an Ice Shard as a Mega Marble will shrink your marble to its regular size instead of freezing you.";
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Ghosts are spooky! Once a ghost collides with you, you will not be attacked by any ghost for about 10-20 seconds.";
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: If a Ghost chases you, go Out of Bounds to reset its spooky timer! It will not go after you for 10-20 seconds after this event.";
+$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: If you're bad at this game, the Out of Bounds special taunt messages will make fun of you.";
```
Comment (thearst3rd) [inline Marble Blast Platinum/platinum/client/scripts/loadinggui.cs:352]: Maybe add along the lines of "This can be disabled in the settings."

Comment (thearst3rd) [reply Marble Blast Platinum/platinum/client/scripts/loadinggui.cs:352]: oh I see, this was moved here from the memes lol

File: Marble Blast Platinum/platinum/client/scripts/loadinggui.cs [hunk 4]
```diff
@@ -415,9 +439,9 @@
 $Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: Checkpoints were coded and added to Marble Blast Platinum two weeks before its first full release to the public.";
 $Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: The original error message noise was in fact Matan screaming 'Spy', referring to Spy47 who originally coded the Online section.";
 $Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: Due to a programming oversight, Marble Blast Platinum used to be able to delete any file or folder from a user's computer, including system files and folders.";
-$Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: Hypercube was an exclusive on Marble Blast Gold for the XBOX. Find it in Ultra's levels as Advanced 21. It retains the Gold marble size and disables the radar.";
+$Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: Before the Xbox section was added to Marble Blast Gold in PlatinumQuest, Hypercube used to be tacked on to the end of Marble Blast Ultra's Advanced section instead, with Gold marble size and no Blast, but Ultra everything else.";
```
Comment (thearst3rd) [inline Marble Blast Platinum/platinum/client/scripts/loadinggui.cs:442]: Currently it still is and it won't be moved out for this update. Let's revert this for now, until it gets moved out later

File: Marble Blast Platinum/platinum/client/scripts/loadinggui.cs [hunk 5]
```diff
@@ -430,7 +454,7 @@
 $Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: The shader model in PlatinumQuest uses Flat Shading with Blinn-Phong specular highlights.";
 $Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: The joystick remapping tip is hard-coded to show up when you first plug in a joystick.";
 $Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: PlatinumQuest's scripts contain over 115,000 lines of code. The extender adds another 50,000 lines. That is almost 4 times as many lines of code as in Marble Blast Gold!";
-$Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: It's been over 10 years and we still don't have a better version of map2dif.";
+$Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: It's been over 15 years and we still don't have a better version of map2dif.";
```
Comment (thearst3rd) [inline Marble Blast Platinum/platinum/client/scripts/loadinggui.cs:457]: Worth mentioning that we have csx2dif

File: Marble Blast Platinum/platinum/client/scripts/loadinggui.cs [hunk 6]
```diff
@@ -449,8 +474,12 @@
 $Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: There are more than 100 ghost sounds! Talk about spooky entertainment!";
 $Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: Winterfest was developed in 3 days! It drained our energy very quickly, as bugs were patched only days after released.";
 $Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: Frightfest was developed a week before its first launch. We learned nothing from the Winterfest event.";
-$Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: The servers, Tsuf and Thistle, are named after Matan's cockatiel and HiGuy's dog, respectively.";
+$Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: PlatinumQuest used to have two dedicated servers, Tsuf and Thistle, which were named after Matan's cockatiel and HiGuy's dog respectively.";
```
Comment (thearst3rd) [inline Marble Blast Platinum/platinum/client/scripts/loadinggui.cs:477]: I'm wondering if we should just remove this one :(

**Pull request merged**